tools/repo

```diff
@@ -173,7 +173,7 @@ getbase()
 			# Assume HBSD always in stable=1: at the moment (2018-01-17) doesn't have
 			# /hardenedbsd-X.Y- url. So, adduct ver version to major version for repo hier
 			_majorver=${ver%%.*}
-			MIRROR="https://fourdots.com/mirror/HardenedBSD"
```

Comment:
Hmm, fourdots.com is no longer a mirror for HBSD base.txz ?

Reply:
It's very unstable, even here in Serbia, and they are Serbian mirror.